app/_layout.tsx

```diff
@@ -9,21 +11,27 @@ import React from 'react';
  *      - Register
  *  - Views: App views once authenticated. The only way to navigate back is to sign out
  *      * Second layer: Uses a navigation drawer, navigating to any of these views replaces the current view. The first layer stack is hidden.
- *          - Todo Lists (exposes another stack in order to edit a specific list)
+ *          - TodoLists (exposes another stack in order to edit a specific list)
  *              - Third layer: Edit stack
- *                * Edit [todo list] can navigate back to Todo Lists
+ *                * Edit [todolist] can navigate back to TodoLists
  *          - SQL Console
  *          - Sign out. Psuedo view to initiate signout flow. Navigates back to first layer.
  */
 const HomeLayout = () => {
+  const system = useSystem();
+  const db = useMemo(() => {
+    return system.powersync;
+  }, []);
   return (
-    <Stack screenOptions={{ headerTintColor: '#fff', headerStyle: { backgroundColor: '#2196f3' } }}>
-      <Stack.Screen name="signin" options={{ title: 'Supabase Login' }} />
-      <Stack.Screen name="register" options={{ title: 'Register' }} />
+    <PowerSyncContext.Provider value={db as any}>
```

Comment:
Why is the `as any` necessary?

Reply:
There were some type mismatches after bumping the packages, but looks to be fine no. Removing `as any`